spec/integration/invite_spec.rb

```diff
@@ -10,6 +10,10 @@
     end
     context 'with a team' do
       let!(:team) { Fabricate(:team, admin_token: 'token') }
+      it "includes a link to the team's slack workspace" do
+        visit "/invite?team_id=#{team.team_id}"
+        expect(page).to have_link(nil, href: team.workspace_url)
```

Comment:
Can we be more specific that the page has something like `If you have previously joined this team, click <a id='team_workspace_url'>here</a> to be redirected.`? Maybe check that the `href` on `team_workspace_url` matches this URL.